build/postbuild.ps1

```diff
@@ -83,18 +101,29 @@ function Copy-OfflineMode()
 	Remove-Item $offlineDrop -Recurse -Force -ErrorAction Ignore
 	New-Item -Type Directory $offlineDrop -ErrorAction Ignore | Out-Null
 
+    Copy-Item "$root\..\.data\catalog.bin" $drop\Microsoft.Fx.Portability.Offline -Recurse -Force
+
 	Copy-Item $drop\Microsoft.Fx.Portability.Offline\$netStandard\* -Include $extensionsToInclude $offlineDrop
 	Copy-Item $drop\Microsoft.Fx.Portability.Reports.Json\$netStandard\* -Include $extensionsToInclude $offlineDrop
 	Copy-Item $drop\Microsoft.Fx.Portability.Reports.Html\$netFramework\* -Include $extensionsToInclude $offlineDrop
 	Copy-Item $drop\ApiPort\$netFramework\* -Include $extensionsToInclude $offlineDrop
 }
 
-Write-Progress -Activity "Creating portability nupkgs" -Status "Complete" -PercentComplete 100
-
-Copy-Item "$PSScriptRoot\..\.data\catalog.bin" $drop\Microsoft.Fx.Portability.Offline -Recurse -Force
-
 Copy-OfflineMode
 
 # Copying the license terms into our drop so we don't have to manually do it when we want to release
-Copy-Item "$PSScriptRoot\..\docs\LicenseTerms" $drop\ApiPort\$netFramework -Recurse -Force
-Copy-Item "$PSScriptRoot\..\docs\LicenseTerms" $drop\ApiPort.Offline\ -Recurse -Force
\ No newline at end of file
+Copy-Item "$root\..\docs\LicenseTerms" $drop\ApiPort\$netFramework -Recurse -Force
+Copy-Item "$root\..\docs\LicenseTerms" $drop\ApiPort.Offline\ -Recurse -Force
+
+if ($PublishVsix) {
+    # Setting these environment variables because they are used in the script
+    # for the upload endpoint:
+    # https://github.com/madskristensen/ExtensionScripts/blob/master/AppVeyor/vsix.ps1#L52-L55
+    # https://github.com/madskristensen/ExtensionScripts/blob/master/AppVeyor/vsix.ps1#L68
+    $env:APPVEYOR_REPO_PROVIDER = "github"
```

Comment:
In [build/Get-BuildTools.ps1:L28-30](https://github.com/conniey/dotnet-apiport/blob/4fe2ceb6a3e87365b1856f417413f58113711ca1/build/Get-BuildTools.ps1#L27-L30), I get it from the github repository and dump it in .tools. Is that what you were asking?